index.html

```diff
@@ -18,29 +20,45 @@
                     <nav class="union">
                         <img src="images/Union.png" alt="menu-image" class="menu-image">
                     </nav>
+                    <nav class="portfolio-options">
+                        <ul class="option-list">
+                            <li><a href="#" class="option-list-items">Portfolio</a></li>
+                            <li><a href="#" class="option-list-items">About</a></li>
+                            <li><a href="#" class="option-list-items">Contact</a></li>
+                        </ul>
+                    </nav>
                 </div>
             </div>
         </div>
     </header>
     <section class="container-headline">
-        <div class="headline">
-            <h1 class="primary-text">
-                I'm Prangon<br>Glad to see you!
-            </h1>
-            <p class="description">
-                I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-            </p>
-            <div class="social-media">
-                <h2 id="media-text">
-                    LET'S CONNECT
-                </h2>
-                <ul class="media-icons">
-                    <li class="icon-list"><a href="#"><img src="images/Twitter.png" alt="Twitter" id="twitter"></a></li>
-                    <li class="icon-list"><a href="#"><img src="images/Linkedin.png" alt="Linkedin" id="linkedin"></a></li>
-                    <li class="icon-list"><a href="#"><img src="images/M.png" alt="M" id="M"></a></li>
-                    <li class="icon-list"><a href="#"><img src="images/GitHub.png" alt="GitHub" id="github"></a></li>
-                    <li class="icon-list"><a href="#"><img src="images/Victory.png" alt="Victory" id="victory"></a></li>
-                </ul>
+        <div class="background">
+            <div class="headline">
+                <h1 class="primary-text">
+                    I'm Prangon<br>Glad to see you!
+                </h1>
+                <p class="description">
+                    I’m a software developer! I can help you build a product, feature or website Look through some of my
+                    work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                    contact me.
+                </p>
+                <div class="social-media">
+                    <h2 id="media-text">
+                        LET'S CONNECT
+                    </h2>
+                    <ul class="media-icons">

```

Comment:
- [ ] I think it would be better to stick with the Figma template you chose; in your design, the background was used differently in the about section. Please, as much as possible, stick with the template you chose.

- ***About section***
 
| Figma design | Your design |
|-------------------|-------------------|
| ![Screenshot from 2022-11-03 22-34-51](https://user-images.githubusercontent.com/65129228/199831680-e61e3dcd-bdfd-42da-88f7-a13e8a3d3c96.png) | ![Screenshot from 2022-11-04 00-34-41](https://user-images.githubusercontent.com/65129228/199840516-b515a596-611d-463e-a551-055877607ae4.png) |

- [ ] Kindly check the background for this section :+1: 

- ***About-myself section***

| Figma design | Your design |
|-------------------|-------------------|
| ![Screenshot from 2022-11-03 22-47-59](https://user-images.githubusercontent.com/65129228/199833690-b01b7646-7414-4891-aca4-b7502393fa99.png) | ![Screenshot from 2022-11-04 00-36-43](https://user-images.githubusercontent.com/65129228/199840990-94042704-e576-4be8-a49d-5bd240a9d4fa.png) | 

- [ ] Please check the desktop version about-myself section on the right side empty. The language, skill, and frame work not present based on the template. Kindly remove those which is not in the desktop version :eagle: 

- ***Contact section***

| Figma design | Your design |
|-------------------|-------------------|
| ![Screenshot from 2022-11-03 22-48-29](https://user-images.githubusercontent.com/65129228/199834017-bd201707-1c8e-4e59-bce2-c484e90659f3.png) | ![Screenshot from 2022-11-04 00-37-03](https://user-images.githubusercontent.com/65129228/199841317-9ce68efc-e346-4d2b-a7d8-b920712b5ffa.png) |

- [ ] I think it would be better to insert a background for the contact section and the title for the section to stick with the Figma template you chose.